utils/mock.js

```diff
@@ -11,15 +11,13 @@ function mockQuery (schema, mockedSchema, parsedQuery, fixture, saveFixture, glo
   let queryVariables = globalQueryVariables || parsedQuery.queryVariables
 
   let mock
-  let mockedQuery
   switch (operationType.toLowerCase()) {
     case 'query':
       // If there are errors defined on the fixture, return them. This should be, the
       // first validation because if it's going to mock an error is because there should be
       // an error, so prevent any extra validation and just return the errors.
-      if (fixture && fixture.errors) {
-        const errors = setFixtureError(fixture.errors)
-        return { errors }
+      if (fixture && fixture.errors && fixture.data === undefined) {
+        return { mockedQuery: { errors: setFixtureError(fixture.errors) } }
```

Comment:
What do you think if we set here this? That way we run `setFixtureError` once and if the data is `null` we don't have to run all the other methods...
```js
  let errors
  switch (operationType.toLowerCase()) {
    case 'query':
      if (fixture && fixture.errors) {
        errors = setFixtureError(fixture.errors)
        if (fixture.data === undefined) {
          return { mockedQuery: { errors } }
        } else if (fixture.data == null) {
          return { mockedQuery: { data: null, errors } }
        }
      }
     .....
```
Doing that we can remove [this](https://github.com/EasyGraphQL/easygraphql-tester/pull/69/files#diff-234c8a3c48360e143b1e839fc7b644e5R5)